--- tilda-0.9.5/src/wizard.c~	2007-12-13 01:24:21.000000000 +0100
+++ tilda-0.9.5/src/wizard.c	2007-12-22 22:36:56.000000000 +0100
@@ -67,7 +67,7 @@
     DEBUG_ASSERT (ltw != NULL);
 
     gchar *window_title;
-    const gchar *glade_file = g_build_filename (DATADIR, "tilda.glade", NULL);
+    const gchar *glade_file = g_build_filename (DATADIR, "tilda/tilda.glade", NULL);
     GtkWidget *wizard_window;
 
     /* Make sure that there isn't already a wizard showing */
--- tilda-0.9.5/Makefile.am~	2007-12-13 01:24:21.000000000 +0100
+++ tilda-0.9.5/Makefile.am	2007-12-22 22:41:31.000000000 +0100
@@ -21,8 +21,6 @@
 Pixmapsdir = ${datadir}/pixmaps
 Pixmaps_DATA = tilda.png
 
-data_DATA = tilda.glade
-
 tilda.desktop: tilda.desktop.in
 	sed -e 's|\@BINDIR\@|$(bindir)|' \
 	    -e 's|\@PIXMAPSDIR\@|$(datadir)/applications|' $< > $@
